arguments-optional.js

```diff
@@ -0,0 +1,79 @@
+// Create a function that sums two arguments together. If only one argument is provided, then return a function that expects one argument and returns the sum.
+
+// For example, addTogether(2, 3) should return 5, and addTogether(2) should return a function.
+
+// Calling this returned function with a single argument will then return the sum:
+
+// var sumTwoAnd = addTogether(2);
+
+// sumTwoAnd(3) returns 5.
+
+// If either argument isn't a valid number, return undefined.
+
+function addTogether() {
+  
+  if (arguments.length > 1 && typeof(arguments[0], arguments[1]) === "number") {
```

Comment:
Why did you not add a number in the parameter brackets?

```js
function addTogether(number1) {
...
}
```

You then don't need the arguments object.  Just test if the `number1` parameter is undefined

Better yet, use a default parameterhttps://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters

```js
if (number1 === null) // do your logic
```